packages/admin-next/dashboard/src/routes/products/product-detail/components/product-variant-section/use-variant-table-columns.tsx

```diff
@@ -140,6 +140,56 @@ export const useProductVariantTableColumns = (
         },
       }),
       ...optionColumns,
+      columnHelper.accessor("inventory", {
+        header: () => (
+          <div className="flex h-full w-full items-center">
+            <span className="truncate">{t("fields.inventory")}</span>
+          </div>
+        ),
+        cell: ({ getValue }) => {
+          const inventory: InventoryItemDTO[] = getValue()
+
+          const hasInventoryKit = inventory.length > 1
+
+          let availableCount = 0
+          const locations = {}
+
+          inventory.forEach((i) => {
+            i.location_levels.forEach((l) => {
+              availableCount += l.available_quantity
+              locations[l.id] = true
+            })
+          })
```

Comment:
This will need to be replaced by `variant.inventory_quantity` when #7701 is merged. The current calculations doesn't take inventory kit combinations into account.